schemas/subscription-results.v1.schema.json

```diff
@@ -0,0 +1,70 @@
+{
+   "$schema":"http://json-schema.org/draft-07/schema#",
+   "$ref":"#/definitions/Main",
+   "definitions":{
+      "Main":{
+         "title":"subscription_results",
+         "type":"object",
+         "additionalProperties":false,
+         "properties":{
+            "version":{
+               "type":"integer"
+            },
+            "payload":{
+               "type":"object",
+               "title":"payload_v3",
+               "additionalProperties":false,
+               "properties":{
+                  "subscription_id":{
+                     "type":"string",
+                     "minLength":1
+                  },
+                  "request":{
+                     "type":"object"
+                  },
+                  "result":{
+                     "type":"object",
+                     "properties":{
+                        "data":{
+                           "type":"array",
+                           "minItems":0,
+                           "items":{
+                              "type":"object",
+                              "minProperties":1,
+                              "additionalProperties":{
+                                 "type":[
+                                    "number",
+                                    "null"
+                                 ]
+                              }
+                           }
+                        }
+                     },
+                     "required":[
+                        "data"
+                     ]
+                  },
+                  "timestamp":{
+                     "type":"string"
```

Comment:
It looks like the specific format of datetime string we send along doesn't match any of the defined formats: https://json-schema.org/understanding-json-schema/reference/string.html#dates-and-times

Reply:
I believe this is `"date-time"`, or at least I am pretty sure what you're sending is RFC 3339. I don't think the examples on that doc page are exhaustive.

Reply:
I had that before and was seeing validation errors :/

Reply:
I think we are sending iso 8601. 3339 is stricter and requires the offset

Reply:
the set of allowed formats in ISO 8601 is vast. Can we change the code to just send RFC 3339? I believe it is a strict subset. But I am fine with `string` as well

Reply:
🤷‍♀️ string is okay with me if it's okay with you too

Reply:
I wouldn't mind taking the opportunity to standardise the way we do datetime... The question is: if snuba starts sending the new format will Sentry's results consumer be able to handle it right away, or is this going to be a painful synchronization process?

Reply:
let's merge string, idk if it's worth fixing rn...

Reply:
sgtm